packages/neo-one-monitor/src/types.js

```diff
@@ -1,5 +1,6 @@
 /* @flow */
 import type { Context } from 'koa';
+import type { Report } from './Reporter';
```

Comment:
On the other hand, the definition of `Report` belongs in this file. `Report` is part of the public api of `Monitor`.

Reply:
One way to think about where things belong is to consider who should "know" about a particular type or class. `Monitor` doesn't need to know about `GaugeMetric`, etc. but it does need to "know" about `Report`, so it belong in this file.